.github/workflows/autobuild.yml

```diff
@@ -31,12 +31,14 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v1
     - name: Compile
-      run: make -j$(sysctl -n hw.logicalcpu)
+      run: |
+        make -j$(sysctl -n hw.logicalcpu)
+        ./mac-bundle.sh
```

Comment:
Oh, that's probably a better way to do multiple commands at once. Again I hate to be so picky, but for consistency I'd rather it just be `run: make -j$(sysctl -n hw.logicalcpu) && ./mac-bundle.sh`. Or if you want to change all the multi-command steps to this format, I'll accept that too :P